src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataClientAutoConfiguration.java

```diff
@@ -7,12 +7,13 @@
 import uk.gov.hmcts.reform.ccd.client.healthcheck.CoreCaseDataHealthIndicator;
 
 @Configuration
+@ConditionalOnProperty(prefix = "core_case_data", name = "api.url")
 @EnableFeignClients(basePackages = "uk.gov.hmcts.reform.ccd.client")
 public class CoreCaseDataClientAutoConfiguration {
 
     @Bean
-    @ConditionalOnProperty(prefix = "core_case_data", name = "api.url")
-    public CoreCaseDataHealthIndicator healthIndicator(CoreCaseDataApi coreCaseDataApi) {
+    public CoreCaseDataHealthIndicator coreCaseData(CoreCaseDataApi coreCaseDataApi) {
```

Comment:
This method name is used as the name displayed in the health check entry -- had to update to make it meaningful.

Reply:
I did not know that, good explanation. Thanks @rlewan